### PR TITLE
Overview concurrency tooltip

### PR DESCRIPTION
**Pull request by Lijiaoa**
- change waiting trial status color to `#FDC401`
- image

### CODE REVIEW

File: src/webui/src/static/const.ts [hunk 1]
```diff
@@ -35,8 +35,10 @@ const COLUMN = ['Trial No.', 'ID', 'Duration', 'Status', 'Default', OPERATION];
 // all choice column !dictory final
 const COLUMNPro = ['Trial No.', 'ID', 'Start Time', 'End Time', 'Duration', 'Status',
 'Intermediate result', 'Default', OPERATION];
+const  CONCURRENCYTOOLTIP = 'TrialConcurrency is the number of trials running concurrently, which is the number of GPUs you want to use if you are setting gpuNum to 1.';
```
Comment (liuzhe-lz) [inline src/webui/src/static/const.ts:38]: Suggest: Trial concurrency is the number of trials running concurrently, which is the number of GPUs you want to use if you set gpuNum to 1.

Comment (Lijiaoa) [reply src/webui/src/static/const.ts:38]: OK!

Comment (QuanluZhang) [reply src/webui/src/static/const.ts]: `which` is a little misleading here.

I think we can just keep the former half: Trial concurrency is the number of trials running concurrently.

Comment (Lijiaoa) [reply src/webui/src/static/const.ts]: I think if only show `Trial concurrency is the number of trials running concurrently.` user still don't understand `concurrency` meaning. Depend on you!

Comment (Lijiaoa) [reply src/webui/src/static/const.ts]: Finally， this tooltip changed to `Trial concurrency is the number of trials running concurrently.`  by Quanlu's comments.

File: src/webui/src/static/style/tableStatus.css [hunk 2]
```diff
@@ -15,7 +15,7 @@
     color:	#EB0716;
 }
 .WAITING{
-    color:	#008B8B;
+    color: yellow;
```
[the remark this answers is not in the document]
Comment (yangou1988) [reply src/webui/src/static/style/tableStatus.css:18]: Please try #FDC401 to display status WAITTING

**Pull request merged**